scripts/backup

```diff
@@ -65,6 +61,18 @@ ynh_psql_dump_db --database="$db_name" > db.sql
 
 ynh_backup --src_path="/etc/logrotate.d/$app"
 
+#=================================================
+# BACKUP THE POSTGRESQL DATABASE
+#=================================================
+ynh_print_info --message="Backing up the PostgreSQL database..."
+
+if [ $database == "pgsql" ]
+then
```

Comment:
```suggestion
then
    ynh_print_info --message="Backing up the PostgreSQL database..."
```
I will include the  `ynh_print_info` in the if statement or have it outside with generique phrasing like `Backing up the database...`

Reply:
I included all `ynh_print_info` in the if statement